llvm/lib/Transforms/IPO/MemProfContextDisambiguation.cpp

```diff
@@ -5573,6 +5579,9 @@ void MemProfContextDisambiguation::performICP(
                                  .getCallee());
         }
         DirectCall.setCalledFunction(TargetToUse);
+        if (MemProfICPNoInlineThreshold &&
```

Comment:
Maybe add a comment here to document why we need to do this?

Reply:
done